tools/iceoryx_build_test.sh

```diff
@@ -69,61 +68,76 @@ while (( "$#" )); do
         CLEAN_BUILD=true
         shift 1
         ;;
-    "release")
-        BUILD_TYPE="Release"
+    "relwithdebinfo")
+        BUILD_TYPE="RelWithDebInfo"
```

Comment:
I was just wondering how to build just as `Release`. Then I remembered you mentioned that would be the default when nothing is specified. Wouldn't it make sense to also have a `release` flag for other who might be confused and hadn't talked to you?